logaggregator/main_test.go

```diff
@@ -0,0 +1,104 @@
+package main
+
+import (
+	"net"
+	"strings"
+	"testing"
+	"time"
+
+	. "github.com/flynn/flynn/Godeps/_workspace/src/github.com/flynn/go-check"
+)
+
+// Hook gocheck up to the "go test" runner
+func Test(t *testing.T) { TestingT(t) }
+
+type S struct {
```

Comment:
Giving this struct a fuller name makes it possible to use with gocheck's '-check.f' flag a la `go test -check.f MyTestSuite`.  (For unit tests it's fairly moot since package specifiers are also handy, but I use this a lot in running the integration tests.)